### PR TITLE
Metrics Aggregate Select

### PR DESCRIPTION
**Pull request by Sami1309**
select different aggregations based on the window sampling for each metrics graph

### CODE REVIEW

File: dashboard/src/components/entitypage/elements/AggregateDropdownSlice.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const aggregateDropdownSlice = createSlice({
+  name: "aggregateDropdown",
+  initialState: {
+    aggregates: ["avg", "avg", "avg"],
+  },
+  reducers: {
+    changeAggregate: (state, action) => {
+      console.log(action.payload);
```
Comment (simba-git) [inline dashboard/src/components/entitypage/elements/AggregateDropdownSlice.js:10]: Logs

File: dashboard/src/components/entitypage/elements/MetricsDropdown.js [hunk 2]
```diff
@@ -57,54 +71,97 @@ const MetricsDropdown = ({ type, name, version, timeRange }) => {
...
                     type={type}
                     name={name}
                     query_type={"latency"}
+                    aggregate={aggregates[1]}
                   />
                 </div>
               )}
 
-              <Typography>Errors per {stepRange}</Typography>
+              <div className={classes.titleBar}>
```
Comment (simba-git) [inline dashboard/src/components/entitypage/elements/MetricsDropdown.js:148]: This breaks out of my screen
image

**Review by simba-git — CHANGES_REQUESTED** [review]
Looks pretty, one CSS bug I found

File: dashboard/src/components/entitypage/elements/MetricsDropdown.js (hunk 2 above)
Comment (Sami1309) [reply dashboard/src/components/entitypage/elements/MetricsDropdown.js:154]: Fixed on latest push

File: dashboard/src/components/entitypage/elements/AggregateDropdownSlice.js (hunk 1 above)
Comment (Sami1309) [reply dashboard/src/components/entitypage/elements/AggregateDropdownSlice.js]: removed

File: dashboard/src/components/entitypage/elements/MetricsDropdown.js [hunk 3]
```diff
@@ -23,11 +23,17 @@ const useStyles = makeStyles((theme) => ({
...
     justifyContent: "space-between",
+    "& > *": {
+      paddingBottom: "0em",
+    },
   },
   aggDropdown: {
     minWidth: "20em",
     display: "flex",
-    padding: theme.spacing(1),
+    //padding: theme.spacing(1),
```
Comment (simba-git) [inline dashboard/src/components/entitypage/elements/MetricsDropdown.js:33]: Remove comment

**Review by simba-git — CHANGES_REQUESTED** [review]

File: dashboard/src/components/entitypage/elements/MetricsDropdown.js (hunk 3 above)
Comment (Sami1309) [reply dashboard/src/components/entitypage/elements/MetricsDropdown.js]: Removed comment

**Review by simba-git — APPROVED** [review]

**Pull request merged**